### ISSUE TITLE
Map appears blank (white screen) when using with Tailwindcss

### ISSUE THREAD

**Issue by danforejtek** [flsy/react-mapycz#172]
I found out that Tailwind [Preflight](https://tailwindcss.com/docs/preflight) css applies styles mentioned below to `img` and `video`

```css
img,
video {
  max-width: 100%;
  height: auto;
}
```

I am not quite sure why this affects map but in order to make it work I just override tailwind's Preflight `max-width: 100%` to `initial` and it works.

```css
.smap {
  img {
    max-width: initial!important;
  }
}
```

Maybe someone who faced same problem will clarify this for me 😄

**Comment by kolebjak** [comment]
Hey, 

thank you for your comment, i'll try to investigate if it's possible to apply some css resets to prevent this.

**Comment by kolebjak** [comment]
@danforejtek I've released version 1.1.2 with your css reset applied. Let me know if that problem occurs even in the new version.
